<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLoadImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadVid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Video</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Background Subtraction</string>
    </property>
    <widget class="QPushButton" name="btnQ1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Background Subtraction</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Optical Flow</string>
    </property>
    <widget class="QPushButton" name="btnQ21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Preprocessing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQ22">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>80</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Video Tracking</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>3. PCA</string>
    </property>
    <widget class="QPushButton" name="btnQ3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>3. Dimension Reduction</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>4. MNIST Classifier Using VGG19</string>
    </property>
    <widget class="QPushButton" name="btnQ41">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Show Model Structure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQ42">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>4.2 Show Acc &amp;&amp; Loss</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQ43">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>4.3 Predict</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQ44">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>4.4 Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>621</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>5. ResNet50</string>
    </property>
    <widget class="QPushButton" name="btnQ5LoadImg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQ51">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>5.1 Show Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQ52">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>5.2 Show Model Structure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQ53">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>5.3 Show Comparison</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQ54">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>5.4 Inference</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
